src/SFA.DAS.EmployerFinance.MessageHandlers/EventHandlers/DummyEventHandler.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using NServiceBus;
+using SFA.DAS.EmployerFinance.Messages;
+
+namespace SFA.DAS.EmployerFinance.MessageHandlers.EventHandlers
+{
+    public class DummyEventHandler : IHandleMessages<DummyEvent>
```

Comment:
This handler can be removed if you prefer, we'll verify that any communication is working as part of the health check ticket.